src/Models/Reminder.cs

```diff
@@ -0,0 +1,65 @@
+using System;
+using Newtonsoft.Json;
+
+namespace StreamChat.Models
+{
+    /// <summary>
+    /// Represents a reminder for a message.
+    /// </summary>
+    public class Reminder
```

Comment:
Reminder is a generic name. If we ever introduce any other form of reminders, we'll run into a naming issue. Perhaps we should name it `MessageReminder`?